spec/requests/api/v1/tokens_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::TokensController, type: :request do
+  describe 'POST#login' do
+    context 'given valid credentials' do
+      it 'returns an access token' do
+        post api_v1_login_path, params: token_request_params
+
+        expect(JSON.parse(response.body)['token_type']).to eq('Bearer')
```

Comment:
We also can use this gem `rspec-json_expectations` to assert the JSON content. Reference: https://github.com/waterlink/rspec-json_expectations